Makefile

```diff
@@ -179,6 +179,14 @@ test-python: pyenv az
 
 admin.kubeconfig:
 	hack/get-admin-kubeconfig.sh /subscriptions/${AZURE_SUBSCRIPTION_ID}/resourceGroups/${RESOURCEGROUP}/providers/Microsoft.RedHatOpenShift/openShiftClusters/${CLUSTER} >admin.kubeconfig
+	$(MAKE) update-admin.kubeconfig
+
+# ! Temporary solution till dns can be migrated to new subscription
+PUBLIC_IP_ADDR = $(shell az network dns record-set a show -g ${RESOURCEGROUP} --zone-name ${LOCATION}.${PARENT_DOMAIN_NAME} -n api.${CLUSTER} --query "aRecords[0].ipv4Address" -o tsv)
+PUBLIC_IP_FQDN = $(shell az network dns record-set a show -g ${RESOURCEGROUP} --zone-name ${LOCATION}.${PARENT_DOMAIN_NAME} -n api.${CLUSTER} --query fqdn -o tsv)
+update-admin.kubeconfig:
```

Comment:
we can merge PR after we remove this portion post DNS migration

Reply:
Thanks Ross! that's the plan